src/components/Sources.js

```diff
@@ -39,7 +39,6 @@ const Sources = React.createClass({
     return dom.div(
       { className: "sources-panel" },
       dom.div({ className: "sources-header" },
-        L10N.getStr("sources.header"),
```

Comment:
We should probably drop this from the strings.json etc. as well unless it is being used somewhere else.